modules/models/src/test/resources/test.yaml

```diff
@@ -50,48 +52,72 @@ deployment:
   # - variable: The value of the variable parameters are defined by the user on the console.
   # The parameters will be used to generate the API of the managed service.
   context:
```

Comment:
i realised that the field name"context" is not clear. Can we think of any other name?

Reply:
varibale?

Reply:
yes, it is not clear enough; 
I think we can create an issue for the new name, leaving it as it is in this PR.

Reply:
I add #195 for this.